python/spec/dependabot/python/update_checker/requirements_updater_spec.rb

```diff
@@ -146,13 +164,13 @@
             context "when needing an update" do
               let(:requirement_txt_req_string) { ">=1.3.0, <1.5" }
 
-              its([:requirement]) { is_expected.to eq(">=1.3.0,<1.6") }
+              its([:requirement]) { is_expected.to eq(">=1.5.0,<1.6") }
```

Comment:
Should Dependabot increase the upper bound simmetrically to the lower bound i.e., leave a 3 major version range in place?

Reply:
I am thinking it shouldn't because you will never know how upcoming versions are handled. Also here we are handling only lower bound the upper bound is working as how it was working before. Do you think we need to reconsider the upper bound in this case?